src/supplemental/nanolib/conf_ver2.c

```diff
@@ -1028,10 +1028,45 @@ conf_bridge_parse_ver2(conf *config, cJSON *jso)
 	return;
 }
 
+#if defined(SUPP_PLUGIN)
+static void
+conf_plugin_parse_ver2(conf *config, cJSON *jso)
+{
+	cJSON *plugin = hocon_get_obj("plugin", jso);
```

Comment:
Maybe use  hocon_get_obj in this way `cJSON *libs = hocon_get_obj("plugin.libs", jso)` is better.

Reply:
Good suggestion, already done